api/rs/slint/Cargo.toml

```diff
@@ -19,7 +19,13 @@ path = "lib.rs"
 
 [features]
 
-default = ["std", "eventloop-winit", "renderer-femtovg", "backend-qt", "compat-0-2-0"]
+default = [
```

Comment:
Did you change anything in this file or are you just reformatting it?

Reply:
Yes, this file got added due to renaming `PlatformAbstraction` to `Platform` in line 78.